modules/swagger-codegen/src/main/resources/csharp/api.mustache

```diff
@@ -80,6 +80,8 @@ namespace {{packageName}}.{{apiPackage}}
     /// </summary>
     {{>visibility}} partial class {{classname}} : {{interfacePrefix}}{{classname}}
     {
+
+        private static Logger log = new Logger(nameof({{classname}}));
```

Comment:
`Logger` needs to be fully qualified here to avoid conflicts with models named `Logger`. Also, I don't know if we'd want to use `nameof` as this is limiting to the compiler the clients may use. I'd consider this a pretty big breaking change.